### PR TITLE
docs: add docs for shutdown op

### PR DESCRIPTION
**Pull request by RushilMehra**
Tests coming soon in a followup PR

### CODE REVIEW

File: src/opcode.rs [hunk 1]
```diff
@@ -1375,6 +1375,7 @@ opcode!(
 // === 5.11 ===
 
 opcode!(
+    /// Shut down all or part of a full duplex connection on a socket, equivalent to `shutdown(2)`.
```
Comment (quininer) [inline src/opcode.rs:1378]: It would be good to add available kernel version in document.

Comment (RushilMehra) [reply src/opcode.rs:1378]: Added, is it worth doing this everywhere? Most docs entries don't have the kernel version explicitly mentioned

Comment (quininer) [reply src/opcode.rs:1378]: Yes, all API should have version docs, just like manpage.

**Comment by quininer** [comment]
Thank you!

**Pull request merged**